apps/sync/index.html

```diff
@@ -11,4 +11,7 @@
        are still in development.
      </p>
   </body>
+  <script src="shared/js/sync/errors.js"></script>
```

Comment:
We can probably lazy load this script as well. It's only needed in case of error.